user-console/src/main/resources/org/pentaho/mantle/public/home/index.jsp

```diff
@@ -118,6 +118,9 @@ if ( PentahoSystem.getApplicationContext().getFullyQualifiedServerURL().toLowerC
       });
 
     });
+	
+	// Set a variable that the inner html pages can read to determine if the ruby theme is active
+	this.isRuby = window.active_theme == "ruby";
```

Comment:
`if(window.active_theme === "ruby") {
  this.css_custom_location = "css/ccp-ruby.css"
}`